ios/Classes/SwiftTruecallerSdkPlugin.swift

```diff
@@ -124,6 +198,23 @@ extension SwiftTruecallerSdkPlugin: TCTrueSDKDelegate {
         eventSink?(map)
     }
     
+    public func verificationStatusChanged(to verificationState: TCVerificationState) {
+        var map = [String: Any]()
+        map[Constants.String.result] = verificationState.rawValue
+        switch verificationState {
+        case .otpInitiated:
+            map[Constants.String.data] = trueSdk.tokenTtl()
+        case .otpReceived,
```

Comment:
I see in case of `otpReceived`, we are not doing anything here. In Android, we auto-read the OTP & pass it on in the callback, so doesn't iOS offer any auto-read functionality?

Reply:
also, in case of `verifiedBefore`, we must pass the `profile` back as the call back data.